utils/urls.js

```diff
@@ -0,0 +1,3 @@
+const { PREDICTION_URL, CURRENT_USER } = require('sharify').data
+
+export const liveAuctionUrl = (id) => `${PREDICTION_URL}/${id}${CURRENT_USER ? '/login' : ''}`
```

Comment:
I think it makes sense to scope by domain! Even just `utils/auctions.js` if we don't want to make too many assumptions about future helper needs.